deployment/stacks/env-config.template.js

```diff
@@ -0,0 +1,17 @@
+window._env_ = {
+  API_REGION: "us-east-1",
```

Comment:
Love the hardcode of us-east-1, so much better than having all these ssm variables and other variables containing the same thing.